Orange/canvas/report/owreport.py

```diff
@@ -337,14 +340,14 @@ def open_report(self):
 
         try:
             report = self.load(filename)
-        except (IOError, AttributeError) as e:
+        except (IOError, AttributeError):
```

Comment:
I tried to open a file with extension .report, but which was not actually a report file. It crashes with `UnpicklingError`. Perhaps you should add it to list, or even catch the general `Exception` here.